pgscatalog_utils/scorefile/combine_scorefiles.py

```diff
@@ -48,6 +64,24 @@ def combine_scorefiles():
                     f"Cannot combine {x} (harmonized to {h.get('HmPOS_build')}) in target build {args.target_build}")
                 raise Exception
 
+        # Build Score header logs
+        pgs_id = h.get('pgs_id')
```

Comment:
We should make this whatever the basename is that being extracted from the file as that's what the accession column is in the scoring file

Reply:
Fixed in the new commit